Issue starting OpenAPI demo

What I'm trying to do

I'm trying to run the OpenAPI demo using
`make run-openapi-demo
`
Actual result
```
hyper@ubuntu:/opt/gopath/src/github.com/did/aries-framework-go$ make run-openapi-demo 
Generating Aries-Framework-Go Test PKI
Signature ok
subject=/C=CA/ST=ON/O=Example Inc.:Aries-Framework-Go/OU=Aries-Framework-Go/CN=*.example.com
Getting CA Private Key
done generating Aries-Framework-Go PKI
Generating and validating controller API specifications using Open API
Generating Open API spec
open /opt/gopath/src/github.com/did/aries-framework-go/build/rest/openapi/spec/openAPI.yml: no such file or directory
Makefile:78: recipe for target 'generate-openapi-spec' failed
make: *** [generate-openapi-spec] Error 1
```

--- a/scripts/generate-openapi-spec.sh
+++ b/scripts/generate-openapi-spec.sh
@@ -7,19 +7,18 @@
 set -e
 
 SPEC_LOC="${SPEC_LOC}"
-SPEC_META="${SPEC_META:-cmd/aries-agent-rest}"
-OUTPUT="$PWD/$SPEC_LOC/openAPI.yml"
+SPEC_DIR="cmd/aries-agent-rest"
+WORKING_DIR="/opt"
+OUTPUT="$SPEC_LOC/openAPI.yml"
 IMAGE="${DOCKER_IMAGE:-quay.io/goswagger/swagger}"
 IMAGE_VERSION="${DOCKER_IMAGE_VERSION:-latest}"
 
-cd $SPEC_META
-
 # generate and validate commands
-GENERATE_COMMAND="generate spec main.go -o $OUTPUT"
+GENERATE_COMMAND="generate spec -w $SPEC_DIR -o $OUTPUT"
 VALIDATE_COMMAND="validate $OUTPUT"
 
 echo "Generating Open API spec"
-docker run --rm -v $HOME:$HOME -w $(pwd) ${IMAGE}:${IMAGE_VERSION} $GENERATE_COMMAND
+docker run --rm -v $(pwd):$WORKING_DIR -w $WORKING_DIR ${IMAGE}:${IMAGE_VERSION} $GENERATE_COMMAND
 
 echo "Validating generated spec"
-docker run --rm -v $HOME:$HOME -w $(pwd) ${IMAGE}:${IMAGE_VERSION} $VALIDATE_COMMAND
+docker run --rm -v $(pwd):$WORKING_DIR -w $WORKING_DIR ${IMAGE}:${IMAGE_VERSION} $VALIDATE_COMMAND

